Task06/script.js

```diff
@@ -0,0 +1,41 @@
+(() => {
+	'use strict';
+	
+	//ドロップダウンの項目のオブジェクトの定義　
+	const dropdownData = {
+		0: 'あくなき探求',
+		1: '不屈の心体',
+		2: '理想への共感',
+		3: '心を動かす',
+		4: '知識を増やす',
+		5: '公明正大'
+	};
+
+	const showEvent = (event) => {
+		event.record.Table.value[0].value.Action5.value = dropdownData[0];
+		for (let count = 1; count < Object.keys(dropdownData).length; count++) {
```

Comment:
オブジェクトの要素数と同じ数分ループするので、`forEach` を使ったほうがわかりやすいかな！
```suggestion
		Object.keys(dropdownData).forEach((key) => {
```
注意：この変更によって、`value: dropdownData[count]` は、`value: dropdownData[key]` に変更する必要が出てくる

配列の要素数と同じ数分のループのときは`forEach`を使うと、
コードを読んだ人が「あ、配列の要素すべてを処理するんだな」ってわかるので少しだけ可読性があがります！
もしくは `for...of` を使ったり。
参考：https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Statements/for...of
`for` は`break`を使って途中でループを抜け出すことができたり、単純に1つずつループしないときなどに使えるかな。

Reply:
本当はforEachでやりたかったのですが、今回テーブルを増やすために、pushをして追加しました。そのため、forEachすると、初期にある０番目の後に配列の数増やしてしまうため、一つ多く作成してしまう問題がありました。
それを解決するために、安直ではありますが、for で一つ飛ばしてforループを回す処理にしました。
１５行目　`event.record.Table.value[0].value.Action5.value = dropdownData[0];` 　はドロップダウンの値を代入しています。

やりたかったことは、
1. 初期テーブルに値を上書き
2. テーブル追加→代入　×５回

この課題でforEachを使うにはどうしたら良いでしょうか？